src/main/java/com/coveros/selenified/OutputFile.java

```diff
@@ -641,12 +696,13 @@ public void finalizeOutputFile(int testStatus) {
             log.error(e);
         }
         // Record the metrics
-        int passes = countInstancesOf("<td class='pass'>Pass</td>");
-        int fails = countInstancesOf("<td class='fail'>Fail</td>");
-        replaceInFile("STEPSPERFORMED", Integer.toString(fails + passes));
+        int passes = countInstancesOf("<td class='pass'>PASS</td>");
+        int fails = countInstancesOf("<td class='fail'>FAIL</td>");
+        int checks = countInstancesOf("<td class='check'>CHECK</td>");
+        replaceInFile("STEPSPERFORMED", Integer.toString(fails + passes + checks));
         replaceInFile("STEPSPASSED", Integer.toString(passes));
         replaceInFile("STEPSFAILED", Integer.toString(fails));
-        if (fails == 0 && errors == 0 && testStatus == 0) {
+        if (fails == 0 && checks == 0 && errors == 0 && testStatus == 0) {
```

Comment:
If you want to be slick here, if fails|checks|errors|testStatus == 0

Reply:
nope, doesn't work...